packages/bot/src/scratch/help-content/flyout-help-base.jsx

```diff
@@ -34,6 +35,7 @@ const HelpBase = ({
                     <Button
                         className='flyout__button-add'
                         has_effect
+                        id={`gtm-${  display_name.replace(/\s/ig, '-')}`}
```

Comment:
@anton-binary this will return same value to `Custom function` in `procedures_callnoreturn` and `procedures_callreturn`. it this we can rename this two?

Reply:
it actually returns different values

Reply:
`gtm-Function` and `gtm-Function-that-returns-a-value` respectively, it'll indeed return the same values for any custom function, but that's ok